### PR TITLE
[#167618914] Show pagopa mail

### PR DESCRIPTION
**Pull request by Edigio**
Added the pago-pa mail field in preferences

In the Preference Screen a field has been added to show the e-mail relating to pago-pa

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Edigio** [comment]
Simulator Screen Shot - iPhone X - 2020-01-21 at 09 38 14

File: locales/en/index.yml [hunk 1]
```diff
@@ -163,6 +163,7 @@ profile:
         enabled: Enabled
         disabled: Disabled
       email: Email address
+      pagoPa_email: PagoPA email
```
Comment (Undermaken) [inline locales/en/index.yml]:
```suggestion
      pagoPa_email: pagoPA email
```

File: locales/it/index.yml [hunk 2]
```diff
@@ -167,6 +167,7 @@ profile:
         enabled: Abilitato
         disabled: Disabilitato
       email: Indirizzo email
+      pagoPa_email: PagoPA email
```
Comment (Undermaken) [inline locales/it/index.yml]:
```suggestion
      pagoPa_email: pagoPA email
```

**Comment by Edigio** [comment]
This is the result, on Android a case where the badge is visible 
<table style="width:100%">
  <tr>
    <th>iOS</th>
    <th>Android</th>
  </tr>
  <tr>
    <td></td>
    <td>
</td>
  </tr>
</table>

**Pull request merged**